ast/ddl.go

```diff
@@ -1223,7 +1346,23 @@ type AlterTableStmt struct {
 
 // Restore implements Node interface.
 func (n *AlterTableStmt) Restore(ctx *RestoreCtx) error {
```

Comment:
add test here for alter table stmt:
https://github.com/pingcap/parser/blob/7b0cfe281af20fb68fe83bd5d66c03c6dcfb6d9d/parser_test.go#L1697-L1785
https://github.com/pingcap/parser/blob/7b0cfe281af20fb68fe83bd5d66c03c6dcfb6d9d/parser_test.go#L2552-L2554

Reply:
Yes, I'll add it here after completing all the restore case.